spec/lib/remote_translations/microsoft/available_locales_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+describe RemoteTranslations::Microsoft::AvailableLocales do
+  describe ".available_locales" do
+    context "app locales that need to be mapping" do
+      it "includes locales with same format that I18n.available_locales" do
+        allow(RemoteTranslations::Microsoft::AvailableLocales).to receive(:remote_available_locales)
+                                                              .and_return(%w[de en es fr pt zh-Hans])
+        expect(RemoteTranslations::Microsoft::AvailableLocales.available_locales.include?("pt-BR")).to eq true
```

Comment:
For readability, we could add a blank line before the expectation.

By the way, what do you think about testing the whole mapping?

```suggestion
        expect(RemoteTranslations::Microsoft::AvailableLocales.available_locales).to eq %w[de en es fr pt-BR zh-CN]
```